.travis.yml

```diff
@@ -12,6 +12,8 @@ before_script:
 - git config --global user.email "test@example.com"
 - git config --global user.name "Test User"
 script:
+- sed -i "s/GOVV_TEST_PACKAGE/$(go list|sed 's/\//\\\//g')/g" ./integration-test/app-different-package/main.go
```

Comment:
I'm trying to understand this line. Why do you need a `GOVV_TEST_PACKAGE` substitution?

Reply:
So that it can run in my project.  I could be missing something fundamental.  Let me re-check.

Reply:
Ahh, integration tests fail on Travis-ci.

Reply:
Tests are fine actually, but why don't you clone the project with the correct package name (i.e. github.com/ahmetb/govvv)? I don't think you're supposed to mutate package names like this.